framework/packages/abstract-app/src/lib.rs

```diff
@@ -13,6 +13,10 @@ pub use error::AppError;
 pub type AppResult<C = Empty> = Result<Response<C>, AppError>;
 mod interface;
 
+// Re-export core abstract utils and types
+pub use abstract_sdk as sdk;
```

Comment:
Not sure about this rename, don't want abstract_sdk compete with some other `sdk`

Reply:
It could have same issues as we had with `core`, consider this:
1. a little bit annoying: 
```rust
use abstract_app::sdk;
use foo_bar::sdk;

fn aloha() {
    sdk::baz();
}
```
2. a little more annoying
```rust
use abstract_app::sdk;
use foo_bar::sdk;

fn aloha() {
    baz(); // LSP suggests to import sdk::baz()
}
```

Reply:
> It could have same issues as we had with `core`, consider this:
> 
>     1. a little bit annoying:
> 
> 
> ```rust
> use abstract_app::sdk;
> use foo_bar::sdk;
> 
> fn aloha() {
>     sdk::baz();
> }
> ```
> 
>     2. a little more annoying
> 
> 
> ```rust
> use abstract_app::sdk;
> use foo_bar::sdk;
> 
> fn aloha() {
>     baz(); // LSP suggests to import sdk::baz()
> }
> ```

Ah nevermind, you're right, it won't let you do this part:
> use abstract_app::sdk;
> use foo_bar::sdk;
because it's under abstract_app